packages/ens-utils/src/registration.ts

```diff
@@ -22,3 +23,60 @@ export type Registration = {
   primaryStatus: PrimaryRegistrationStatus;
   secondaryStatus: SecondaryRegistrationStatus | null;
 };
+
+export const getDomainRegistration = (
+  expiryTimestamp: Timestamp,
+): Registration => {
+  const primaryStatus = getPrimaryRegistrationStatus(expiryTimestamp);
+  const secondaryStatus = getSecondaryRegistrationStatus(expiryTimestamp);
+  return {
+    expiryTimestamp,
+    primaryStatus,
+    secondaryStatus,
+    registrationTimestamp: null,
```

Comment:
It seems that we must not be using this field anywhere since we never are passing in a `registrationTimestamp` into the `getDomainRegistration` function. Therefore suggest removing this field from `Registration`.

Reply:
That's right, this was raised here: https://blockful.slack.com/archives/C03JQ5URTEG/p1720654856342799

One story was created by you to achieve this goal: https://app.shortcut.com/ps-web3/story/25399/remove-duplicate-expirytimestamp-field

Sounds like a separate PR goal, wdyt?

Reply:
@FrancoAguzzi Sure that's fine :+1: